app/components/map-from-id.js

```diff
@@ -73,20 +73,38 @@ export default Component.extend({
 
   @computed('mapConfig.layers')
   builtLayers(layers = []) {
-    return layers.map((layer) => {
+    const builtLayers = [];
+
+    layers.forEach((layer) => {
       if (layer.type === 'choropleth') {
         const { id, source, paintConfig } = layer;
-        return {
+
+        builtLayers.push({
           id,
           type: 'fill',
           source,
           'source-layer': layer['source-layer'],
           paint: buildPaint(paintConfig),
-        };
-      }
+        });
 
-      return layer;
+        // for choropleth fill layers, push an outlines line layer as well
+        builtLayers.push({
+          id: `${id}-line`,
```

Comment:
It only applies if the type is `choropleth`, which will always be a polygon geometry.  I do think it's a good fit for the build step, it is basically turning our custom choropleth config into two mapboxGL layers, something that we should do the same way with every choropleth...

I could see it abstracted way into an `addLineLayer: true` property + a utility, so it can be added easily wherever it is needed.